confluent_kafka/avro/__init__.py

```diff
@@ -53,8 +53,7 @@ def __str__(self):
 
 
 from confluent_kafka.avro.cached_schema_registry_client import CachedSchemaRegistryClient
-from confluent_kafka.avro.serializer import (SerializerError,
-                                             KeySerializerError,
+from confluent_kafka.avro.serializer import (KeySerializerError,
```

Comment:
hi, OK, I will revert this. I only removed it because the SerializerError is not used in the __init__.py script. I saw it as unnecessary import..